1/Dockerfile.rhel7

```diff
@@ -47,14 +47,16 @@ ADD ./contrib/s2i /usr/libexec/s2i
 # NOTE: When adding new Jenkins plugin, you have to create the symlink for the
 # HPI file created by rpm to /opt/openshift/plugins folder.
 RUN mkdir -p /opt/openshift/plugins && \
-    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.hpi && \
-    ln -s /usr/lib64/jenkins/credentials.hpi /opt/openshift/plugins/credentials.hpi && \
+    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.jpi && \
+    ln -s /usr/lib64/jenkins/credentials.hpi /opt/openshift/plugins/credentials.jpi && \
     # Prefer the latest version of the credentials plugin over the bundled
     # version.
-    touch /opt/openshift/plugins/credentials.hpi.pinned && \
-    ln -s /usr/lib64/jenkins/durable-task.hpi /opt/openshift/plugins/durable-task.hpi && \
-    ln -s /usr/lib64/jenkins/kubernetes.hpi /opt/openshift/plugins/kubernetes.hpi && \
+    touch /opt/openshift/plugins/credentials.jpi.pinned && \
+    ln -s /usr/lib64/jenkins/durable-task.hpi /opt/openshift/plugins/durable-task.jpi && \
+    ln -s /usr/lib64/jenkins/kubernetes.hpi /opt/openshift/plugins/kubernetes.jpi && \
     /usr/local/bin/fix-permissions /opt/openshift && \
+    /usr/local/bin/fix-permissions /opt/openshift/plugins && \
+    chmod -R g+rw /opt/openshift/plugins/*.jpi && \
```

Comment:
that this appears to have worked means something is going wrong w/ the fix-permissions script..

in fact given that we're making the call on /opt/openshift, adding another call on /opt/openshift/plugins also shouldn't be necessary.

I wonder if chmod/chgrp -R doesn't follow symlinks or something.

that appears to be the case:
```
zahir:/tmp$ ln -s /tmp/b a/blink
zahir:/tmp$ ls -l a
total 0
lrwxrwxrwx. 1 bmp bmp 6 Jun 12 23:24 blink -> /tmp/b

zahir:/tmp$ ls -l b
-rw-rw-r--. 1 bmp bmp 0 Jun 12 23:24 b

zahir:/tmp$ chmod -R a+rwx /tmp/a
zahir:/tmp$ ls -l b
-rw-rw-r--. 1 bmp bmp 0 Jun 12 23:24 b

zahir:/tmp$ cd /tmp/a
zahir:/tmp/a$ chmod a+rwx *
zahir:/tmp/a$ ls -l /tmp/b
-rwxrwxrwx. 1 bmp bmp 0 Jun 12 23:24 /tmp/b
```

http://ubuntuforums.org/showthread.php?t=2046341

we may need to do this entirely via a "find" operation.